src/BlockChain/docs/latest-2019-04-30/Get-AzBlockchainMember.md

```diff
@@ -0,0 +1,173 @@
+---
+external help file:
+Module Name: Az.Blockchain
+online version: https://docs.microsoft.com/en-us/powershell/module/az.blockchain/get-azblockchainmember
+schema: 2.0.0
+---
+
+# Get-AzBlockchainMember
+
+## SYNOPSIS
+Get details about a blockchain member.
+
+## SYNTAX
+
+### List1 (Default)
+```
+Get-AzBlockchainMember [-SubscriptionId <String[]>] [-DefaultProfile <PSObject>] [<CommonParameters>]
+```
+
+### Get
+```
+Get-AzBlockchainMember -Name <String> -ResourceGroupName <String> [-SubscriptionId <String[]>]
+ [-DefaultProfile <PSObject>] [<CommonParameters>]
+```
+
+### GetViaIdentity
+```
+Get-AzBlockchainMember -InputObject <IBlockchainIdentity> [-DefaultProfile <PSObject>] [<CommonParameters>]
+```
+
+### List
+```
+Get-AzBlockchainMember -ResourceGroupName <String> [-SubscriptionId <String[]>] [-DefaultProfile <PSObject>]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+Get details about a blockchain member.
+
+## EXAMPLES
+
+### Example 1: {{ Add title here }}
+```powershell
+PS C:\> {{ Add code here }}
```

Comment:
Same comment - applies across the board to all examples - we should have at least one workign example for each cmdlet